src/helpers/inStockTv.js

```diff
@@ -0,0 +1,10 @@
+
+let inStockCounter = 0;
+function inStockTv (){
+    for (let i = 0; i < inventory.originalStock.length; i++) {
```

Comment:
Hier doe de .length methode op een integer, dat is niet mogelijk. Je wilt het doen op de array zelf, dus inventory.length